.github/workflows/build.yml

```diff
@@ -266,3 +266,56 @@ jobs:
       with:
         name: integration-test-results-db2-${{ matrix.java }}
         path: integration-test-results
+  db2-migration:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 'openjdk8' ]
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up OpenJDK
+      uses: joschi/setup-jdk@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - name: Build samples
+      run: mvn -B install --file fhir-examples --no-transfer-progress
+    - name: Build parent without tests
+      run: |
+        mvn -B install --file fhir-parent -DskipTests -P integration --no-transfer-progress
+    - name: Server Integration Tests
+      env:
+        # debian-based linux uses C.UTF-8 by default and Derby doesn't like that
+        LC_ALL: en_US.UTF-8
+      run: |
+        export WORKSPACE=${GITHUB_WORKSPACE}
```

Comment:
you could probably map this in env to be consistent with the prior step

Reply:
its set this way in all the jobs in this build.yml
I think I had some issue setting it from the env section, but I really don't remember...will leave as-is unless you want it changed